src/NuGet.Core/NuGet.Common/Errors/NuGetLogCode.cs

```diff
@@ -222,6 +222,11 @@ public enum NuGetLogCode
         /// </summary>
         NU1301 = 1301,
 
+        /// <summary>
+        /// HTTP Source specified.
```

Comment:
```suggestion
        /// Insecure HTTP Source specified.
```

Without context of this PR, just browsing the list of error codes, it sounds normal to specify HTTP sources, so it's not obvious why this would be a warning or error.